packages/@okta/vuepress-site/docs/reference/api/users/index.md

```diff
@@ -317,6 +318,79 @@ curl -v -X POST \
 ##### Response Example
 
 
+```json
+{
+  "id": "00ub0oNGTSWTBKOLGLNR",
+  "status": "ACTIVE",
+  "created": "2013-07-02T21:36:25.344Z",
+  "activated": null,
+  "statusChanged": null,
+  "lastLogin": null,
+  "lastUpdated": "2013-07-02T21:36:25.344Z",
+  "passwordChanged": "2013-07-02T21:36:25.344Z",
+  "profile": {
+    "firstName": "Isaac",
+    "lastName": "Brock",
+    "email": "isaac.brock@example.com",
+    "login": "isaac.brock@example.com",
+    "mobilePhone": "555-415-1337"
+  },
+  "credentials": {
+    "password": {},
+    "provider": {
+      "type": "IMPORT",
+      "name": "IMPORT"
+    }
+  },
+  "_links": {
+    "activate": {
+      "href": "https://${yourOktaDomain}/api/v1/users/00ub0oNGTSWTBKOLGLNR/lifecycle/activate"
+    },
+    "self": {
+      "href": "https://${yourOktaDomain}/api/v1/users/00ub0oNGTSWTBKOLGLNR"
+    }
+  }
+}
+```
+
+#### Create User with Password Import Inline Hook
+
+
+Creates a user with a specified [password hook](#password-hook-object). See [Password Import Inline Hook](/docs/reference/password-hook/) for more details.
+The new user is able to login immediately after activation with the specified password.
```

Comment:
Might be good to add some more explanation of how this works, e.g.; "The Password Inline Hook is triggered to handle verification of the end user's password the first time the user tries to log in. Okta calls the Password Inline Hook to check that the password the user supplied is valid. If the password is valid, Okta stores the password that was provided and can authenticate the user independently from then on."

Reply:
Ok, will do.